### PR TITLE
[RDF] Import a Pandas data frame to RDataFrame

### PR DESCRIPTION
**Pull request by peremato**
# This Pull request:

## Changes or fixes:
- Simplifies the importing of Pandas data frames to RDataFrame. 
- Added tutorial `df035_RDFFromPandas.py` to demonstrate this functionality

## Checklist:

- [ x] tested changes locally
- [ ] updated the docs (if necessary)

This PR fixes # 
none

### CODE REVIEW

**Comment by peremato** [comment]
@phsft-bot build just on ROOT-centos7/default, ROOT-ubuntu14/default

**Comment by peremato** [comment]
@phsft-bot build just on ROOT-centos8/default, ROOT-ubuntu2204/default

**Comment by peremato** [comment]
@dpiparo can you review it? It fails again on Windows only (this time more test failing)
I checked and the new tutorial runs successfully on the other platforms.

**Review by dpiparo — APPROVED** [review]
LGTM!

File: bindings/pyroot/pythonizations/python/ROOT/_facade.py [hunk 1]
```diff
@@ -306,6 +306,15 @@ def MakeNumpyDataFrameCopy(np_dict):
                 return MakeNumpyDataFrame(np_dict)
 
             ns.FromNumpy = MakeNumpyDataFrameCopy
+
+            # make a RDataFrame from a panda dataframe
+            def MakePandaDataFrame(df):
```
Comment (guitargeek) [inline bindings/pyroot/pythonizations/python/ROOT/_facade.py:311]: It's called "pandas", so can we maybe call it `MakePandasDataFrame`?

Sorry for only chiming in now, I was unaware of the PR before it got merged.

Comment (dpiparo) [reply bindings/pyroot/pythonizations/python/ROOT/_facade.py:311]: Apologies for not noticing and not adding you to the loop. Addressed here https://github.com/root-project/root/pull/14990

Comment (peremato) [reply bindings/pyroot/pythonizations/python/ROOT/_facade.py:311]: The function is not visible, what is visible is `ROOT.RDF.FromPandas` which is consistent with `ROOT.RDF.FromNumpy`

Comment (dpiparo) [reply bindings/pyroot/pythonizations/python/ROOT/_facade.py:311]: yes, indeed it's internal! I applied a minimal change in #14990 anyway it was very easy :)

**Pull request merged**